website/integrations/services/proxmox-ve/index.md

```diff
@@ -24,36 +24,63 @@ The following placeholders will be used:
 -   `proxmox.company` is the FQDN of the Proxmox VE server.
 -   `authentik.company` is the FQDN of the authentik install.
 
-### Step 1
+## authentik setup
 
-Under _Providers_, create an OAuth2/OpenID provider with these settings:
+1. **Navigate to the Admin interface, then click Providers to create an OAuth2/OpenID provider with these settings:**
 
--   Name: proxmox
--   Redirect URI: `https://proxmox.company:8006` (Note the absence of the trailing slash, and the inclusion of the webinterface port)
--   Signing Key: Select any available key
+    - **Name:** proxmox
+    - **Redirect URI:** `https://proxmox.company:8006` (No trailing slash, include the web interface port)
+    - **Signing Key:** Select any available key
 
-### Step 2
+2. **Create an application using this provider**
```

Comment:
```suggestion
2. Create an application using the provider.
```

Reply:
I see the aesthetic/readability appeal of bolding the whole sentence, but... we don't do that in any other docs...